pulley/src/opcode.rs

```diff
@@ -26,7 +26,7 @@ macro_rules! define_opcode {
 
         impl Opcode {
             /// The value of the maximum defined opcode.
-            pub const MAX: u8 = define_opcode!( @max $( $name )* ) + 1;
+            pub const MAX: u8 = Opcode::ExtendedOp as u8;
```

Comment:
This change was needed because otherwise we overran the recursion limit and rustc asked for a `#![recursion_limit = "bigger"]` value which I wanted to avoid so I switch this to using the final enum here instead.

Reply:
Might be an off by one here? Does this need a `+ 1`?

Reply:
Double-checked, but this doesn't have the `+1` for now. (it's at least the same as before)